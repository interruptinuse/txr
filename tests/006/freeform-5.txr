@(next :list (append (repeat (list (mkstring 99 #\A)) 100)
                     '("FOOBARXYZZY" "NEXT" "NEXTNEXT")))
@(freeform)
@(skip)FOO@{bar}XY
@zzy
@next
@nextnext
