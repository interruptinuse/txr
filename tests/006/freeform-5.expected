bar="BAR"
zzy="ZZY"
next="NEXT"
nextnext="NEXTNEXT"
